pkg/client/client.go

```diff
@@ -261,3 +262,54 @@ func MergeAnnotations(toolchainObject client.Object, newAnnotations map[string]s
 	}
 	toolchainObject.SetAnnotations(annotations)
 }
+
+// ApplyUnstructuredObjects applies the given Unstructured objects on the cluster.
+func ApplyUnstructuredObjects(ctx context.Context, cl client.Client, unstructuredObjects []*unstructured.Unstructured, newLabels map[string]string) (bool, error) {
+	applyClient := NewApplyClient(cl)
+	anyApplied := false
+
+	for _, unstructuredObj := range unstructuredObjects {
+		var object = unstructuredObj.DeepCopyObject().(client.Object)
+		if err := runtime.DefaultUnstructuredConverter.FromUnstructured(unstructuredObj.Object, object); err != nil {
+			return false, err
+		}
```

Comment:
What do we need this for? What does this do?

Reply:
Yeah, this is a left over 🤦‍♂️ 

Removed https://github.com/codeready-toolchain/toolchain-common/pull/373/commits/b1606562340e1711c01b821400d1bb1e55e06c9d

thanks